src/test/java/school/hei/haapi/integration/CommentIT.java

```diff
@@ -51,12 +35,32 @@ void setUp() {
     setUpS3Service(fileService, student1());
   }
 
+  @Test
+  void manager_read_comments_without_filter_ok() throws ApiException {
+    ApiClient apiClient = anApiClient(MANAGER1_TOKEN);
+    CommentsApi api = new CommentsApi(apiClient);
+
+    List<Comment> actual = api.getComments(1, 10, null);
+
+    assertEquals(comment3(), actual.get(0));
+  }
+
+  @Test
+  void teacher_read_comments_with_ascendant_filter_ok() throws ApiException {
+    ApiClient apiClient = anApiClient(TEACHER1_TOKEN);
+    CommentsApi api = new CommentsApi(apiClient);
+
+    List<Comment> actual = api.getComments(1, 10, ASC);
+
+    assertEquals(comment1(), actual.get(0));
```

Comment:
these tests can be merged, differentiate the expected and actual values by their naming